.gitignore

```diff
@@ -1,3 +1,9 @@
 /.output
 /PSModules
-/.vscode
\ No newline at end of file
+/.vscode

```

Comment:
I think you can collapse `/.output`, `.vscode`, and `/.vargrant` to `.*` so Git just ignores all "hidden" files/directories.

Reply:
Is `.github` needed in VCS, though?

Reply:
Updated, but kept .github from being filtered out.

Reply:
`.gitignore` only prevents `git status` from showing items and `git remove` from deleting things. You can still `git add` something that matches an ignore pattern and Git will happily add it. Once added, .gitignore never applies to an item.